src/network/networkd-address-generation.c

```diff
@@ -117,6 +117,8 @@ static void generate_stable_private_address_one(
                 siphash24_compress(&link->hw_addr.infiniband[INFINIBAND_ALEN - 8], 8, &state);
         else
                 siphash24_compress(link->hw_addr.bytes, link->hw_addr.length, &state);
+        if (link->ssid)
+                siphash24_compress_string(link->ssid, &state);
```

Comment:
Please add a newline before and after the added logic

Reply:
Ok, added them.